src/commands/api/validate.js

```diff
@@ -0,0 +1,46 @@
+const BaseCommand = require('../../support/command/base-command')
+const { getApiIdentifierArg } = require('../../support/command/parse-input')
+const { getApi } = require('../../requests/api')
+const { pipeAsync } = require('../../utils/general')
+const { getResponseContent } = require('../../support/command/handle-response')
+
+class ValidateCommand extends BaseCommand {
+  async run() {
+    const { args } = this.parse(ValidateCommand)
+    const apiPath = getApiIdentifierArg(args)
+    const validationResult = await this.getValidationResult(apiPath)
+    // eslint-disable-next-line immutable/no-let
+    let hasCritical = false
+    const validationResultsStr = validationResult.validation
+          .map(err => {
+            if (err.severity === 'CRITICAL') hasCritical = true
+            return `${err.line}: \t${err.severity} \t${err.description}`
+          })
+          .join('\n')
+    if (validationResult.validation.length) this.log('line \tseverity \tdescription\n')
+    this.log(validationResultsStr)
+    
+    if (hasCritical) this.exit(1)
+    this.exit(0)
```

Comment:
```suggestion
    this.exit(hasCritical ? 1 : 0)
```

Reply:
I think the if () {} style is more readable here.

Reply:
I'm not a fan of multiple `this.exit()` from a control flow perspective. Perhaps an alternative is:
```
const statusCode = hasCritical ? 1 : 0
this.exit(statusCode)
```

Reply:
Agreed (to both of ye!)
```js
if(hasCritical)
  return this.exit(1) // We need to return out of here
return this.exit(0)
```
Why?
a) Early returns are readable
b) Chasing variables can be a pain, if we can use constants, let us.
c) We shouldn't call `this.exit` twice

Reply:
@ponelat What's your suggested style for this?
a) I wouldn't call the second last line of the function an "early return". That's more appropriate for returning early as a result of invalid input for example.
b) I was wondering if we could do an alternative implementation that returns a `const hasCritical`, perhaps by doing a separate `Array.some()` on the validation result.

Reply:
@barryadk 
a) You're right, there isn't any code that follows. I was looking at it, in isolation. Although I still prefer the "early-not-really-early" return, since it's easier to write code between those two statements in the future. But this is minor, I won't push for either option too strongly.
b) I wouldn't overthink the `let/const` syntax, unless there is a readability issue. In this particular case, it doesn't bother me either way, the readability could be improved with `const isCritical = Array.some(v => v.severity === 'CRITCAL'` at the cost of doing going through the loop twice.